### ISSUE TITLE
How to type the # character?

### ISSUE THREAD

**Issue by Jacalz** [micmonay/keybd_event#33]
Hi. I tried to type the "#" key by using "kb.HasShift(true)" + "keybd_event.VK_3" which is how I would do it on my Swedish keyboard. How can I get the "#" typed? I can't figure it out so any help would be appreciated :)

**Comment by Jacalz** [comment]
Hmm, this worked fine on Windows, but not on Linux when I tested it. As the main use case for the software is to run on Windows, this isn't a huge problem.

**Comment by micmonay** [comment]
Hi, it's depends of your keyboard layout. If you test `shift+3` on linux hardware keyboard you have `#` char ?

**Comment by Jacalz** [comment]
Yes, both have the same Nordic QWERTY layout where shift + 3 gives a #.
